website/index.html

```diff
@@ -1,102 +1,236 @@
-    <header>
-      <div id="logo-container">
-        <h1 id="logo-text" class="text-center">E2E Test Dapp</h1>
-        <img id="mm-logo" src="metamask-fox.svg"/>
-      </div>
-    </header>
-    <section>
-      <h3 class="card-title">Status</h3>
-      <div class="row">
-        <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
-          <p class="info-text alert alert-primary">Network: <span id="network"></span></p>
-        </div>
-        <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
-          <p class="info-text alert alert-secondary">ChainId: <span id="chainId"></span></p>
+  <body>
+    <main class="container-fluid">
+      <header>
+        <div id="logo-container">
+          <h1 id="logo-text" class="text-center">
+            E2E Test Dapp
+          </h1>
+
+          <img id="mm-logo" src="metamask-fox.svg" />
         </div>
-        <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
-          <p class="info-text alert alert-success">Accounts: <span id="accounts"></span></p>
+      </header>
+      <section>
+        <h3 class="card-title">
+          Status
+        </h3>
+
+        <div class="row">
+          <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
+            <p class="info-text alert alert-primary">
+              Network: <span id="network"></span>
+            </p>
+          </div>
+
+          <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
+            <p class="info-text alert alert-secondary">
+              ChainId: <span id="chainId"></span>
+            </p>
+          </div>
+
+          <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
+            <p class="info-text alert alert-success">
+              Accounts: <span id="accounts"></span>
+            </p>
+          </div>
         </div>
+      </section>
 
-      </div>
-    </section>
-
-    <section>
-      <div class="row d-flex justify-content-center">
-        <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
-          <div class="card">
-            <div class="card-body">
-              <h4 class="card-title">Basic Actions</h4>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="connectButton" disabled></button>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="getAccounts">eth_accounts</button>
-              <p class="info-text alert alert-secondary">eth_accounts result: <span id="getAccountsResult"></span></p>
+      <section>
+        <div class="row d-flex justify-content-center">
+          <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12">
+            <div class="card">
+              <div class="card-body">
+                <h4 class="card-title">
+                  Basic Actions
+                </h4>
+
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="connectButton"
+                  disabled
+                >
+                  Connect
```

Comment:
Why was `Connect` added here? This was removed because we'd determine during the page load what to display here (depending on whether they were connected and whether MetaMask was installed)

Reply:
That still works, I just figured it was nice and potentially informative to have a default.

Reply:
Sure, I suppose so